client/my-sites/plan-features-2023-grid/index.tsx

```diff
@@ -394,24 +388,17 @@ export class PlanFeatures2023Grid extends Component<
 	}
 
 	renderTabletView() {
-		const { planProperties, hideFreePlan } = this.props;
-		const plansToShow = hideFreePlan
-			? [ TYPE_PERSONAL, TYPE_PREMIUM, TYPE_BUSINESS, TYPE_ECOMMERCE, TYPE_ENTERPRISE_GRID_WPCOM ]
-			: [
-					TYPE_FREE,
-					TYPE_PERSONAL,
-					TYPE_PREMIUM,
-					TYPE_BUSINESS,
-					TYPE_ECOMMERCE,
-					TYPE_ENTERPRISE_GRID_WPCOM,
-			  ];
+		const { planProperties } = this.props;
+		const plansToShow = planProperties
+			.filter( ( { isVisible } ) => isVisible )
```

Comment:
I decided to add the `.filter( ( { isVisible } ) => isVisible )` to each location to make is more visible where the filtration is happening.